src/main/java/com/alkemy/ong/controller/MemberController.java

```diff
@@ -35,10 +39,18 @@
     }
 
     @PreAuthorize("hasRole('ROLE_ADMIN')")
-    @GetMapping
-    public ResponseEntity<List<MemberDto>> listMembers(){
-        return ResponseEntity.ok(memberService.listAllMembers());
+    @GetMapping()
+    public ResponseEntity<List<MemberDto>> listMembers(@RequestParam Map<String ,Object> params){
+
+        Integer page = params.get("page") != null ? (Integer.valueOf(params.get("page").toString()) -1) : 0;
```

Comment:
Esta lógica que pones acá debería esstar en el service, recibís el número de página por parámetro (en este caso vos lo planteas con **params**) y le pasas eso al service, allí es el lugar para realizar las operación del PageRequest.of(). Y si le pasas el map entonces la page la recuperás también en el service, es decir lo que tenés en la línea 45.